devicemodel/include/vhost_vsock.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2018 Intel Corporation.
+ * Copyright (C) 2022 Intel Corporation.
```

Comment:
These two lines should be combined into one if both were from Intel Corporation.